language/move-prover/tests/sources/functional/ModifiesErrorTest.move

```diff
@@ -0,0 +1,102 @@
+address 0x0 {
```

Comment:
It's very nice when you can see at a glance whether an error in an .exp file is intended.  We've done this in the past by adding _incorrect to function names.  It seems like the function name doesn't appear in the error messages above, though (should they?).  If you add a comment after each error line // incorrect, maybe that will make it more clear that the error is intentional?

Reply:
I have added a suffix "_incorrect" to names of functions on which some error is reported.